requirements/base.txt

```diff
@@ -11,3 +11,9 @@ logstash-formatter==0.5.9
 uwsgi
 requests
 postcodeinfo>=0.0.2,<0.1.0
+postcodes
+django-extensions
+werkzeug
+mock
+django-nose
+
```

Comment:
django-extensions, mock and django-nose should probably go in to a separate file